packages/palette/src/elements/TimeRemaining/TimeRemaining.tsx

```diff
@@ -47,7 +52,11 @@ export const TimeRemaining: React.SFC<{
 
   return (
     <Flex flexDirection="column" alignItems="center">
-      <Sans size="3" color={highlight} weight="medium">
+      <Sans
+        size={timerFontSize ? timerFontSize : "3"}
```

Comment:
```
timerFontSize || "3"
```
Buit this isn't needed because this can be hoisted up to where the variable is being destructured from props, as an optional value.